components/docs-chef-io/content/automate/High_Availability.md

```diff
@@ -0,0 +1,301 @@
+
+DR cluster can be implemented on Chef Automate through a regular backup and restore measures syncing the data from the production cluster to the
+DR cluster. Typically, these two clusters are located in different data centers or cloud provider regions. This is intended to provide an environment that can be made production in a short period with minimal data loss.
+
+# System & Software Requirements 
+
+## For platform support 
+
++----------------------+-----------------------------------------------+
+|      Operating       |                               Tested          |
+| Systems              | Versions                                      |
++======================+===============================================+
+| Red Hat Enterprise   |  7, 8 (For 8 or above                         |
+| Linux                | versions, **SELinux** configuration must be   |
+|                      | permissive. By default, in RHEL               |
+| (64 Bit OS)          | 8, **SELinux** configuration is enforced).    |
+|                      |                                               |
+|                      |                                               |
+|                      |                                               |
+|                      | Red Hat Enterprise Linux derivatives include  |
+|                      | Amazon Linux v1 (using RHEL 6 packages) and   |
+|                      | v2 (using RHEL 7 packages).                   |
++----------------------+-----------------------------------------------+
+| Ubuntu (64 Bit OS)   |  14.04.x, 16.04.x, 18.04.x, 20.04.x           |
++----------------------+-----------------------------------------------+
+| Centos (64 Bit OS)   |  7                                            |
++----------------------+-----------------------------------------------+
+
+## Virtual Machine (VM) instances Type 
+
++--------------+-------------+--------------------------+--------------+
+|              |       Type  |                    RAM   |              |
+| Instance     |             |                          | Volume-size  |
++==============+=============+==========================+==============+
+| PostgreSQL   | t3a.medium  | 4GB RAM For Test and 8G  | 50GB         |
+|              |             | For Prod                 | (dedicated   |
+|              |             |                          | hard disk    |
+|              |             |                          | space need   |
+|              |             |                          | to be        |
+|              |             |                          | assigned to  |
+|              |             |                          | '/')         |
++--------------+-------------+--------------------------+--------------+
+| El           | m5a.large   | 8GB RAM For Test and 16G | 50G          |
+| asticsearch  |             | For Prod                 | B (dedicated |
+|              |             |                          | hard disk    |
```

Comment:
it should be m5.large instead of m5a.large

Reply:
done